i18n/en_GB/messages/messages.json

```diff
@@ -464,6 +464,27 @@
   "donation_confirmation_survey_title": "What almost stopped you from donating?",
   "donation_confirmation_survey_content": "Your response helps us to improve our service and our website.",
   "donation_confirmation_survey_link_text": "Give feedback",
-  "donation_confirmation_survey_link": ""
+  "donation_confirmation_survey_link": "",
+
+  "site_name": "{pageTitle} - Wikimedia Deutschland e V.",
+  "access_denied_page_title": "Access denied",
+  "address_update_success_page_title": "Address updated",
+  "comment_list_page_title": "Comment list",
+  "comment_ticker_page_title": "Comment ticker",
+  "contact_form_page_title": "Contact form",
+  "donation_confirmation_page_title": "Donation confirmed",
+  "donation_form_page_title": "Donation form",
+  "error_page_title": "Error",
+  "faq_page_title": "Frequently Asked Questions",
+  "membership_application_confirmation_page_title": "Membership application confirmed",
+  "membership_application_page_title": "Membership application form",
+  "page_not_found_page_title": "Page not found",
+  "privacy_protection_page_title": "Privacy protection",
+  "subscription_confirmation_page_title": "E-Mail address confirmed",
+  "supporters_page_title": "Hall of Fame",
+  "static_page_title": "Static Page",
```

Comment:
Is this shown for every static page? Or is it replaced by an actual page title at a later step?

Reply:
Sorry, this is a leftover code. I will remove it.
Now, we set the title for static pages dynamically.